SU-SWS/stanford_media
# D8CORE-2431: adding height field to google form.

# READY

# Summary
- Here we add an extra field to the Google Form media type, to hold a height value for the iframe.
- We update the field formatter to display it correctly
- We add the necessary configs and tests
- We add the configuration form necessary as well.

# Needed By (Date)
- 12/3

# Urgency
- 4/10

# Steps to Test

1. Make sure you have a nice clean cardinalsites stack.
2. Update your composer.json: `"su-sws/stanford_profile": "dev-D8CORE-2431 as dev-8.x-1.x`
3. `composer update`
4. `blt di -n`
5. Make a new piece of content with a text area.  In the text area, embed a google form.
6. Notice you'll be prompted for a height value when you save the media.
7. Look in the media library, and try to update the google form.  Notice the height value works.

# Affected Projects or Products
All these use stanford_media as dependencies:
- stanford_person
- stanford_events
- stanford_text_editor
- stanford_news

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
